### DISCUSSION TITLE
How to add options to the client

### DISCUSSION THREAD

**Discussion by rizdaputra** [grpc/grpc-web#1459] [Q&A - Errors / Debugging Help]
I seem to be encountering a problem where i receive incomplete response on the client because the size of the response from the server is too big. How do i increase the max receive size in grpc client generated with grpc-web? i tried adding 
`{ "grpc.max_receive_message_length": 1000000}` in initialization of the client but it doesnt seem to resolve the issue. The error i get in the client is incomplete response

**Comment by sampajano** [comment]
@rizdaputra Hi thanks for the question..

I'm not familiar with this issue in particular..

I wonder what proxy are you using? I'm wondering if you are able to identify if it's an issue with the proxy or the server itself.

If you are able to make a test rpc call using a gRPC client (e.g. https://github.com/grpc/grpc-node) without proxy, you might be able to identify where the issue is at, which can give a clue on what to do next.

Hope this helps :)
